### PR TITLE
Migrate to Micronaut application plugin

### PR DESCRIPTION
**Pull request by yuri-sergiichuk**
This PR is part of the efforts required to enable GraalVM Native Image support.

While the compilation to the Native Image is now blocked by the unsupported APIs usage in the framework, we still may benefit from the migration to the new Micronaut plugin.

The migration-related work is done in the https://github.com/SpineEventEngine/chat-bot/tree/add-graalvm-support branch.

### CODE REVIEW

File: buildSrc/src/main/kotlin/deps.kt [hunk 1]
```diff
@@ -74,12 +69,14 @@ object Spine {
     const val server = "io.spine:spine-server:${Versions.spineCore}"
     const val client = "io.spine:spine-client:${Versions.spineCore}"
     const val core = "io.spine:spine-core:${Versions.spineCore}"
+    const val serverTestUtil = "io.spine:spine-testutil-server:${Versions.spineCore}"
```
Comment (alexander-yevsyukov) [inline buildSrc/src/main/kotlin/deps.kt:72]: I think this dependency comes together with `spine.enableJava().server()`. Please check.

**Review by alexander-yevsyukov — APPROVED** [review]
LGTM, with one minor request.

File: google-chat-bot/build.gradle.kts [hunk 2]
```diff
@@ -38,28 +42,30 @@ spine {
...
 }
 
-dependencies {
-    annotationProcessor(enforcedPlatform(Deps.build.micronaut.bom))
-    annotationProcessor(Deps.build.micronaut.injectJava)
-    annotationProcessor(Deps.build.micronaut.validation)
-
-    compileOnly(enforcedPlatform(Deps.build.micronaut.bom))
+micronaut {
+    runtime(MicronautRuntime.NETTY)
```
Comment (dmdashenkov) [inline google-chat-bot/build.gradle.kts:46]: Should we consider doing this with Bootstrap?

**Review by dmdashenkov — APPROVED** [review]
@yuri-sergiichuk, LGTM.

File: google-chat-bot/build.gradle.kts (hunk 2 above)
Comment (yuri-sergiichuk) [reply google-chat-bot/build.gradle.kts:46]: Configuring the Micronaut runtime? I don't think so. It supports 9 runtimes as of now and for me, it is definitely not Bootstrap's responsibility.

**Pull request merged**